packages/itwin/grouping-mapping-widget/src/widget/utils.ts

```diff
@@ -0,0 +1,7 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+export type CreateTypeFromInterface<Interface> = {
```

Comment:
why???

Reply:
This allows the interfaces that were generated by the swagger codegen client to be used in the itwinui-react type generics.

Reply:
Will disappear once we get a more established reporting client.

Reply:
I played with this locally, should be able to replace all usage of Mapping type except for these two spots: 
![image](https://user-images.githubusercontent.com/11051042/152217776-508f5027-c680-4373-b75b-a35283028d26.png)

I see its because Table expects a Record 
![image](https://user-images.githubusercontent.com/11051042/152217980-f13ba9b9-f0e7-40ae-bc34-cf37e158fb49.png)

You should document this, I see it was initially copied from the demo-portal